packages/trader/src/Modules/Trading/Components/Form/screen-small.jsx

```diff
@@ -59,16 +63,25 @@ const CollapsibleTradeParams = ({
                     <BarrierMobile />
                 </div>
             )}
+            {isVisible('strike') && (
+                <div collapsible={'true'}>
+                    <Strike />
+                </div>
+            )}
             <MobileWidget is_collapsed={is_collapsed} toggleDigitsWidget={toggleDigitsWidget} />
             {has_allow_equals && <AllowEqualsMobile collapsible='true' />}
             {is_multiplier && (
                 <div collapsible='true'>
                     <RiskManagementInfo />
                 </div>
             )}
-            <div className='purchase-container'>
+            {is_vanilla ? (
                 <Purchase />
-            </div>
+            ) : (
+                <div className='purchase-container'>
```

Comment:
can we use here `className={!is_vanilla ? 'purchase-container' : ''}` instead of additional condition with <Purchase/> 
if it doesn’t affect styles for vanilla

Reply:
In that case it would create an extra empty block around, which is better to remove to improve semantics